src/Web/XHR/FormData.js

```diff
@@ -4,30 +4,30 @@ exports["new"] = function () {
   return new FormData();
 };
 
-exports._fromFormElement = function(form) {
+export function _fromFormElement(form) {
```

Comment:
Are we wanting to standardize on this, or just use whatever the `lebab` output is? For my part I'm perfectly fine just using whatever `lebab` uses (it's not like we ever took a standard approach anyway).

Reply:
I'm just using whatever `lebab` output is. It does output `export var` rather than `export const`, so I have to fix that after the fact, but I'm not doing anything about these functions.